lib/screens/authentication/recover/recover_account_found/interactor/recover_account_found_bloc.dart

```diff
@@ -23,12 +24,12 @@ class RecoverAccountFoundBloc extends Bloc<RecoverAccountFoundEvent, RecoverAcco
   final AuthenticationBloc _authenticationBloc;
   StreamSubscription<int>? _tickerSubscription;
 
```

Comment:
Note: I removed the integer from ticks - it will get cancelled when we hit recovery time

Having it limited to the number of seconds makes me think it might stop too early (like 1 second too early) or do something else that's weird when the app is paused . 

We don't need the ticker to know how many seconds are remaining, the state knows that.